tests/services/test_crash_reports.py

```diff
@@ -64,10 +64,23 @@ def test_clear(crash_manager):
             assert crash_manager.afc.stat(path)['st_birthtime'] > test_dir_birth_time
 
 
-def test_pull(crash_manager):
+def test_pull(crash_manager, delete_test_dir):
     crash_manager.afc.makedirs(PATH_COMPONENT)
     dir_list = crash_manager.ls(depth=-1)
     crash_manager.pull(BASENAME)
     pulled_list = [file[len(BASENAME):] for file in glob.glob(f'{BASENAME}/**', recursive=True)][1:]  # ignore root path
     assert sorted(dir_list) == sorted(pulled_list)
     shutil.rmtree(BASENAME)
+
+
+@pytest.mark.parametrize(
+    ("message", "expected"),
+    (
+        ("sysdiagnose (full) complete", True),
+        ("Sysdiagnose completed. File path: /foo/bar.tar.gz", True),
```

Comment:
this one fails. should it be decapitilized?

Reply:
Thanks, my mistake. I was making some last minute changes and forgot to rerun.